webcompat/db/__init__.py

```diff
@@ -70,9 +80,32 @@ class Site(SiteBase):
     ranking = Column(Integer)
 
     def __init__(self, url, priority, country_code, ranking):
+        """Initialize the topsite db parameters."""
         self.url = url
         self.priority = priority
         self.country_code = country_code
         self.ranking = ranking
 
 SiteBase.metadata.create_all(bind=site_engine)
+
+
+IssuesBase = declarative_base()
+IssuesBase.query = issues_db.query_property()
+
+
+class Issues(IssuesBase):
+    """SQLAchemy base object for Issues."""
+
+    __tablename__ = 'issues'
+
+    issue_number = Column(Integer, primary_key=True)
+    title = Column(String)
+    is_issue = Column(Boolean, default=True)
+
+    def __init__(self, issue_number, title, is_issue):
+        """Initialize the topsite db parameters."""
+        self.issue_number = issue_number
+        self.title = title
+        self.is_issue = is_issue
```

Comment:
I guess it would be good to spend time being thoughtful about what info we would need in an issues db to avoid unnecessary migrations in the future. Would we want labels? Milestones? Something else?